packages/react/src/card/card.tsx

```diff
@@ -29,7 +29,7 @@ export const CardMediaImg = forwardRef<HTMLImageElement, CardImageMediaProps>(
     return (
       <Component
         {...rest}
-        className={clsx("hds-card__media__img", className as undefined)}
+        className={clsx("hds-card__media--img", className as undefined)}
```

Comment:
This is not correct BEM notiation. the `--` is [reserved for modifiers](https://getbem.com/naming/#modifier), not children. As long as the `img` is nested under a `media` it needs to be underscore

<img width="775" alt="image" src="https://github.com/user-attachments/assets/73c19021-0a7d-47f5-9676-e543a6350423" />

Looks like some people have opionons about bem grandchildren
https://scalablecss.com/bem-nesting-grandchild-elements/

Reply:
<img width="966" alt="image" src="https://github.com/user-attachments/assets/8499162b-4c06-4b5a-9180-bee1c282aecd" />

Reply:
Too narrow validation rules?

Reply:
👍